core/templates/dev/head/css/oppia.css

```diff
@@ -3960,7 +3960,7 @@ md-card.preview-conversation-skin-supplemental-card {
   display: inline-block;
   float: right;
   margin-top: 3%;
-  width: 67%;
+  width: 65%;
```

Comment:
@WickedBrat, have you tested this (twice or thrice)? (It's not working)
Also, I think for the mobile view you should work on ` @media(max-width: 629px)` (Line 3996), although I'm not sure whether changing the width will work here! 
PTAL!

Reply:
@DubeySandeep Actualy the problem occurs only for tablet size on the main build and testserver build. So as I said above 
> @seanlip Considering the fact that everything works fine for mobile and desktop site in the main build. I'll do the required fix for tablet size.

I made this fix only for tablet size and not for mobile size. 
Yes, I've tested this several times, it works fine for me on the tablet size.

Reply:
So to clarify my comment before, the learner dashboard should work on all devices.

Reply:
Yeah, my point is that, it already works for mobile and desktop sites, and I made it work for tablet sites as well

Reply:
Hm. So when @DubeySandeep says it's not working, what does he mean?

Reply:
@WickedBrat & @seanlip, In #4252 it's explicitly mentioned that the issue is with the mobile view and this PR isn't resolving the issue for mobile view. Although it's working fine on test and prod site but it's not working on dev site (i.e, develop branch).

Reply:
In that case, as the issue fixed for main build and teset server, and don't need fix. I think we can close this issue and file a new one for tablet size devices.
Suggestion @DubeySandeep @seanlip

Reply:
I'm mostly confused at this point. The learner dashboard should work for mobile, tablet and desktop on the test server, main site and in the develop branch. If this isn't the state of affairs, then a fix is needed.

In particular it's important that the develop branch be fully fixed, since that's what's going to be cut in the next release.